src/ui/src/app/modules/user/pages/teacher-view/question-edit-view/question-edit-view.component.html

```diff
@@ -1,10 +1,10 @@
-<h1 mat-dialog-title>New Question</h1>
+<h1 mat-dialog-title class="header">Edit Question</h1>
```

Comment:
Ich dachte wir haben kein Edit funktion

Reply:
What would be your suggested approach to edit existing questions / which title would you suggest?

Reply:
So we Remove it ?

Reply:
ye

Reply:
The solutuion for editing a question is to delete the question and add a new correct one. 
Not that userfriendly but it is easier then editing the json file whole the time.